Dockerfile

```diff
@@ -66,24 +66,27 @@ RUN chmod 644 $PROJECTOR_DIR/ide/projector-server/lib/*
 
 FROM eclipse-temurin:${containerJdkVersion}-jdk-jammy
 
-# Add custom CA certs
+# Add custom CA certificates
 ARG extraCaCertsDir
 ADD ${extraCaCertsDir} /usr/local/share/ca-certificates/
 
 RUN true \
 # Any command which returns non-zero exit code will cause this shell script to exit immediately:
-   && set -e \
+    && set -e \
 # Activate debugging to show execution details: all commands will be printed before execution
-   && set -x \
+    && set -x \
+    && update-ca-certificates \
 # install packages:
     && apt-get update \
 # packages for awt:
     && apt-get install libxext6 libxrender1 libxtst6 libxi6 libfreetype6 -y \
 # packages for user convenience:
+    && apt-get install git bash-completion vim -y \
+ # packages and CA certs added for CCCS use (general use) 
+    && apt-get install ca-certificates jq vim -y \ 
+    && update-ca-certificates \ 
+    && apt-get install sudo unzip zip sed apt-utils -y \
     && apt-get install git bash-completion sudo -y \
```

Comment:
This could be one line or at least one apt-get statement. Also sudo is there twice.

Reply:
Je vais ajuster.

Reply:
Done